src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,41 @@
+package seedu.addressbook.commands;
+
+/**
+ * Sorts all persons in the address book to the user.
+ */
+public class SortCommand extends Command{
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts all persons either by name or phone "
+            + "Example: " + COMMAND_WORD + " name\n"
+            + "Example: " + COMMAND_WORD + " phone\n";
+
+    public static final String MESSAGE_SUCCESS = "List sorted";
+
+    private static final String SORT_BY_NAME_COMMAND_WORD = "name";
+    private static final String SORT_BY_PHONE_COMMAND_WORD = "phone";
+
+    private String sortBy = "";
+
+    public SortCommand(String sortBy) {
+        this.sortBy = sortBy;
+    }
+
+    @Override
+    public CommandResult execute() {
+        switch(sortBy) {
+
+        case SORT_BY_NAME_COMMAND_WORD:
+            addressBook.sortByName();
+            break;
+        case SORT_BY_PHONE_COMMAND_WORD:
+            addressBook.sortByPhone();
+            break;
+        default:
+            new HelpCommand();
```

Comment:
Even though this is the 'default' case, it is still recommended to include a break or "// Fallthrough" comment if you decide not to add the break. Refer to the coding standard.